src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/avoidescapedunicodecharacters/Example3.java

```diff
@@ -0,0 +1,28 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="AvoidEscapedUnicodeCharacters">
+      <property name="allowByTailComment" value="true"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.avoidescapedunicodecharacters;
+
+// xdoc section -- start
+public class Example3 {
+  // OK, a normal String below
+  String unitAbbrev = "μs";
+  // violation below
+  String unitAbbrev2 = "\u03bcs";
+  String unitAbbrev3 = "\u03bcs"; // it is  μs
```

Comment:
Let's keep above comments in all examples. To make sure code is same except for comments.
Only comments are changing from Example to Example , and such comment explains why it is ok or it is violation.

Reply:
```
 // OK
  String unitAbbrev = "μs"; // a normal String below
  // violation below
  String unitAbbrev2 = "\u03bcs"; //because not all characters are escaped ('s')
```

This is the way we can do it else all the other ways you were suggesting like /```
/ violation below because printable escape character
  String unitAbbrev1 = "\u03bcs";
```
showing the same error of expected and final output differs. But this was the way it can be done . And I feel that it also satisfies the need that comments are varying in every example and above lines are same i.e `violation` or `ok`

Reply:
> // OK

We do not allow simple ok comment, as it has no much value to users. Better to do ` // ok , here is some reason  why `

>// violation below

This is good only if it is very obvious violations, sometimes we put reason like `// violation below because printable escape character` it helps users easier to understand why violation happens.

Reply:
> then we would need to add `// violation `in the below line . Like this `//violation below due to this reason` and then in then end of the below line `// violation` to sort the error of unexpected elements.

Reply:
Please rephrase or update your comment , I can not understand it.

Reply:
While running `mvn clean verif`y there is an error occuring that says unexpected elements are there in` Example1.java` that differ with expected value. And to resolve that issue, we need to add a `//violation` line in the end of the violated java statement like this 
```
// violation below, not all characters are escaped ('s').
  String unitAbbrev2 = "\u03bcs"; // violation
```

then only I am able to pass the checks. You suggest me any other way if possible. I can implement. above line gives the reason for violation and below `// violation` line is just to pass the check

Reply:
Please investigate why it works for https://github.com/checkstyle/checkstyle/blob/93f3a0c82b7cefe0116943fff20a82984e0b34d2/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/visibilitymodifier/Example10.java#L40

More examples at https://github.com/search?q=repo%3Acheckstyle%2Fcheckstyle%20violation%20below&type=code

Reply:
Violation comment can on same line or above or below.

Reply:
I made the changes, now i think things should be fine @romani and sorry that you had to explain things to me.